backend/capellacollab/sessions/hooks/interface.py

```diff
@@ -55,6 +57,10 @@ class ConfigurationHookRequest:
     provisioning: list[sessions_models.SessionProvisioningRequest]
     project_scope: projects_models.DatabaseProject | None
     session_id: str
+    authentication_information: tuple[
+        users_models.DatabaseUser, tokens_models.DatabaseUserToken | None
+    ]
+    global_scope: permissions_models.GlobalScopes
 
```

Comment:
auth info and scope are missing from the docs

Reply:
Replaced auth info with pat since it's a bit more trivial and added it to the docs.